en/firefox/browsers/history/what-is-a-browser.ftl

```diff
@@ -0,0 +1,48 @@
+
+# Variables:
+#   $tools (url) - link to https://blog.mozilla.org/firefox/internet-search-engine-browser/
+what-is-a-browser-its-important = It’s important that everyone has access to the web, but it’s also vital that we all <a href="{ $tools }">understand the tools</a> we use to access it. We use web browsers like { -brand-name-mozilla } { -brand-name-firefox }, { -brand-name-google } { -brand-name-chrome }, { -brand-name-microsoft } { -brand-name-edge } and { -brand-name-apple } { -brand-name-safari } every day, but do we understand what they are and how they work?
+
+what-is-a-browser-in-a-short-period-long = In a short period of time we’ve gone from being amazed by the ability to send an email to someone around the world, to a change in how we think of information. It’s not a question of how much you know anymore, but simply a question of what browser or app can get you to that information fastest.
+what-is-a-browser-in-a-short-period = In a short period of time we’ve gone from being amazed by the ability to send an email to someone around the world, to a change in how we think about information.
+what-is-a-browser-how-does-a-web = How does a web browser work?
+what-is-a-browser-a-web-browser-long = A web browser takes you anywhere on the internet. It retrieves information from other parts of the web and displays it on your desktop or mobile device. The information is transferred using the Hypertext Transfer Protocol, which defines how text, images and video are transmitted on the web. This information needs to be shared and displayed in a consistent format so that people using any browser, anywhere in the world can see the information.
+
+# Variables:
+#   $standards (url) - link to https://developer.mozilla.org/docs/Archive/Web_Standards
+what-is-a-browser-sadly-not-all = Sadly, not all browser makers choose to interpret the format in the same way. For users, this means that a website can look and function differently. Creating consistency between browsers, so that any user can enjoy the internet, regardless of the browser they choose, is called <a href="{ $standards }">web standards</a>.
+
+# Variables:
+#   $html (url) - link to https://developer.mozilla.org/docs/Glossary/HTML
+what-is-a-browser-when-the-web-browser = When the web browser fetches data from an internet connected server and it then uses a piece of software called a rendering engine to translate that data into text and images. This data is written in <a href="{ $html }">Hypertext Markup Language</a> (HTML) and web browsers read this code to create what we see, hear and experience on the internet.
```

Comment:
The sentence structure is not correct, `When... and blah then blah blah`. May I suggest to add a comma and remove `and` and `then` so it reads: `When the web browser..., it uses a piece...`